Assets/MRTK/SDK/Experimental/NonNativeKeyboard/Scripts/NonNativeKeyboardTouchAssistant.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    public class NonNativeKeyboardTouchAssistant : MonoBehaviour
```

Comment:
Might be good to make this script run at edit time. No need to add these components at runtime.

Reply:
You might not always want this, and this way it's easy to disable it.